_episodes/13-conditionals.md

```diff
@@ -24,58 +24,65 @@ keypoints:
     *   Body containing one or more statements is indented (usually by 4 spaces)
 
 ~~~
-mass = 3.54
-if mass > 3.0:
-    print(mass, 'is large')
+animal = "dog"
+if animal == "dog":
+    print(animal, 'is fluffy')
 
-mass = 2.07
-if mass > 3.0:
-    print (mass, 'is large')
+animal = "snake"
+if animal == "dog":
+    print (animal, 'is fluffy')
 ~~~
 {: .language-python}
 ~~~
-3.54 is large
+dog is fluffy
 ~~~
 {: .output}
 
 ## Conditionals are often used inside loops.
 
 *   Not much point using a conditional when we know the value (as above).
 *   But useful when we have a collection to process.
-
+*   We can also use the in operator on strings directly.
 ~~~
-masses = [3.54, 2.07, 9.22, 1.86, 1.71]
-for m in masses:
-    if m > 3.0:
-        print(m, 'is large')
+animals = ["aardvark", "bat", "cat", "dog"]
+acount = 0
+for animal in animals:
+   for letter in animal:
+      if letter == 'a':
+         acount += 1
+print("Count of letter a:", acount)
 ~~~
 {: .language-python}
 ~~~
-3.54 is large
-9.22 is large
+Count of letter a: 5
 ~~~
 {: .output}
 
+*   We can also use the in operator on strings directly.
+~~~
+sentence = "Is there a bat in here?"
+if "bat" in sentence:
+   print("There's a bat!")
+~~~
 ## Use `else` to execute a block of code when an `if` condition is *not* true.
```

Comment:
```suggestion
~~~
{: .language-python}

~~~
There's a bat!
~~~
{: .output}


## Use `else` to execute a block of code when an `if` condition is *not* true.
```
Need to add the language tag for the code block and then would be good to have the output too.